src/main/kotlin/ru/mipt/npm/gradle/KScienceCommonPlugin.kt

```diff
@@ -112,7 +111,7 @@ open class KScienceCommonPlugin : Plugin<Project> {
                     }
                 }
 
-                sourceSets.invoke {
+                sourceSets {
```

Comment:
For some reason it was not working before

Reply:
🤔